app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -267,8 +269,25 @@ def pod_full_name
     "#{options[:pod_namespace]}/#{options[:pod_name]}"
   end
 
+  def inspector_admin_secret
+    kubeclient = kubernetes_client
+    begin
+      inspector_sa = kubeclient.get_service_account(IMAGE_INSPECTOR_SA, INSPECTOR_NAMESPACE)
+      # TODO: support multiple imagePullSecrets. This depends on image-inspector support
+      return nil unless inspector_sa.try(:imagePullSecrets)
+      return (inspector_sa.imagePullSecrets.collect &:name)[0]
```

Comment:
@simon3z This is a preparation to support multiple secrets. I will just remove the ```[0]``` and add in line 338 an ```.each```. Do you want me to remove it for now ?